<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRotate</class>
 <widget class="QMainWindow" name="frmRotate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Antenna properties</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblThe_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate about X axis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRotate_z">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate about Y axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate about Z Axis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRotate_x">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRotate_y">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblThe_start_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOrigin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblThe_start_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Waveform</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sinusoid</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtAngel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblThe_start_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Angel-XY/YZ/XZ(deg)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtScale">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAxisLength">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis Length(m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblThe_start_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Radiation Import</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRadioFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkShowAxis">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thickness(pixel)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtThickness">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
